backend/src/Service/Notification/NotificationLocalService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Service\Notification;
+
+use App\AutoMapping;
+use App\Entity\NotificationLocalEntity;
+use App\Manager\Notification\NotificationLocalManager;
+use App\Request\Notification\NotificationLocalCreateRequest;
+use App\Response\Notification\NotificationLocalResponse;
+use App\Constant\Notification\NotificationConstant;
+
+class NotificationLocalService
+{
+    public function __construct(private AutoMapping $autoMapping, private NotificationLocalManager $notificationLocalManager)
+    {
+    }
+
+    /**
+     * @param integer $userId
+     * @param string $title
+     * @param string $text
+     * @param integer $orderId
+     * @return NotificationLocalResponse
+     */
+    public function createNotificationLocal($userId, $title, $text, $orderId = null): NotificationLocalResponse
```

Comment:
We may define the parameters types in the functions definition instead of the annotations.